x-pack/test/apm_api_integration/tests/observability_overview/observability_overview.ts

```diff
@@ -54,7 +54,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         expectSnapshot(response.body.serviceCount).toMatchInline(`9`);
 
-        expectSnapshot(response.body.transactionPerMinute.value).toMatchInline(`64.8`);
+        expectSnapshot(response.body.transactionPerMinute.value).toMatchInline(`302.2`);
```

Comment:
This seems like too big a difference to be right. 

I'm wondering how trustworthy the metric documents in the archive are. Perhaps some improvements were made to metric doc generation after we built the archive? 

@dgieselaar any idea?

Reply:
is this the bug we've found with _doc_count?

Reply:
Not sure. Has that been fixed and would rebuilding the archives fix this?

Reply:
It's an es bug on the search side, so rebuilding the archive won't matter. Not sure what the status is of the bug fix.

Reply:
It's fixed and backported to 7.x 11 days ago so I assumed it made 7.14

Reply:
It's a missing filter query, it aggregates over all metric documents, not just the transaction metrics one. I've pushed a fix.